src/nvim/autocmd.c

```diff
@@ -700,6 +701,7 @@ int check_ei(void)
 // buffer loaded into the window.  "what" must start with a comma.
 // Returns the old value of 'eventignore' in allocated memory.
 char_u *au_event_disable(char *what)
+  FUNC_ATTR_PURE
```

Comment:
I can't help but think that this suggestion is bogus. Not only does it call functions not marked as pure (potentially breaking purity), but it doesn't suggest these functions to be marked pure either. It also directly uses `strcpy`, which doesn't use `__attribute__ ((__pure__))`, as opposed to `strlen`. Only including because I'd like input from someone more knowledgable about how the attribute works.

Reply:
I think I agree with you.

❌